package.json

```diff
@@ -19,7 +19,7 @@
     "schema.json"
   ],
   "engines": {
-    "node": ">=16.7.0"
+    "node": ">=14.0.0"
```

Comment:
`fs.rm` was added in node 14.14 so I think that should be noted as the minimum. There were others like top level await in 14.8, and some import issues until 14.13, but I found 14.14 was lowest minor version that would pass the tests.

Reply:
Oh, great catch! Thank you for running the tests!

Done.